### DISCUSSION TITLE
Change text of AddNew Button for single select cases once an Image has been selected to "Change…"

### DISCUSSION THREAD

**Discussion by leepeterson** [pods-framework/pods#6764] [Ideas]
## Issue Overview
Even when "Single File" is selected for the "Upload Limit" on a "Field Type" of "File/Image/Video", the resulting button labeled "Add File/Image/Video" does not grey out or take on a `disabled` attribute, allowing the user to effectively "Replace File/Image/Video".

## Expected Behavior
"Add File/Image/Video" button should disable itself after a file/image/video is uploaded, and only if "Single File" is specified in "Upload Limit".

## Current Behavior
The button labeled "Add File/Image/Video" does not grey out or take on a `disabled` attribute, allowing the user to effectively "Replace File/Image/Video".

## Possible Solution
"Add File/Image/Video" button should take on an attribute of `disabled` if the aforementioned details hold true.

## Steps to Reproduce (for bugs)
1. Create Pod, create "Field Type" of "File/Image/Video"
2. Within the "Additional Field Options" tab, choose "Single File" in "Upload Limit"
3. Save Pod, view Pod, upload a file/image/video
4. "Add File/Image/Video" button continues to allow for "adding more file/image/video files" when in fact it's actually "replacing the file/image/video file"

## Todos
- [ ] Tests
- [ ] Documentation

## WordPress Environment
<details>
Pods 2.7.1
WordPress Version: 4.9.5
PHP Version: 7.2.5
MySQL Version: 5.7.22
Server Software: Apache/2.4.33 (Unix) OpenSSL/1.0.2g
</details>

**Comment by quasel** [comment]
Good idea!
